### ISSUE TITLE
Problem with planned and remaining hours

### ISSUE THREAD

**Issue by davinci41** [projectexpert/pmis#17]
The problem appears after editing the task view by the PMI modules
Creating a new task and completing Planned Hours does not recalculate.

![image](https://cloud.githubusercontent.com/assets/25403547/24727553/b021af34-1a56-11e7-837f-8e9e4a1689c6.png)

![image](https://cloud.githubusercontent.com/assets/25403547/24727562/b6b3d39a-1a56-11e7-9da3-611f80eecc54.png)

After entering the time spent in the task, neither the working time nor the working time are not counted
![image](https://cloud.githubusercontent.com/assets/25403547/24727576/c5116ad8-1a56-11e7-87c6-6b672b6f4c5e.png)

**Comment by sysadminmatmoz** [comment]
I can't reproduce the problem.
Setting the planned schedule:
![screenshot_2017-04-07_13-05-33](https://cloud.githubusercontent.com/assets/6734798/24797849/6c1a44fc-1b93-11e7-99fa-5cc5f46fa4c7.png)
Save.
Register the work done:
![screenshot_2017-04-07_13-10-41](https://cloud.githubusercontent.com/assets/6734798/24797919/b719c6a8-1b93-11e7-9757-4f8f46c9ccac.png)
Everything is as expected:
![screenshot_2017-04-07_13-12-35](https://cloud.githubusercontent.com/assets/6734798/24797947/e3b0e7b4-1b93-11e7-807e-fd6ad02b2ac0.png)

**Comment by davinci41** [comment]
When creating a new task, the remaining time automatically replaces the scheduled time.

I do not complete the remaining time, what more as typing the hours worked it is the remaining time I appear in the minus.

I am using the version available on odoo.com

https://www.youtube.com/watch?v=KgTiGeep8eg

https://www.youtube.com/watch?v=b_hCW__ZVag

https://www.youtube.com/watch?v=lrZtXTH9Rf4

**Comment by sysadminmatmoz** [comment]
The remaining time is the difference between the planned time and worked time, so it's normal, that with a new task, where the worked time is still 0, the remaining time results the same as the planned time.

The remaining time is not meant to be edited manually except for the reasons of corrections (example: we initially planned less time than needed, so we correct the remaining time when and if we review the task). Don't touch the remaining time field on newly created task.

**Comment by davinci41** [comment]
In the default time module, the time worked on the remaining time is redesigned.
Https://www.youtube.com/watch?v=TFp4LuCmxTM

After installing PMI differently to the department:
1. I do not understand why after entering the overworked time the remaining time is in the negative. (After 2 hours of work, 2 hours remaining)
Https://www.youtube.com/watch?v=KgTiGeep8eg
2. I do not understand why it does not change or change without appearing at scheduled times.
https://www.youtube.com/watch?v=b_hCW__ZVag

**Comment by sysadminmatmoz** [comment]
The activity scheduling considers the activity duration field and not the "initially planned hours" since at time it was designed without considering the use of the settings/project option "Manage time estimation on tasks". So to mimic the same functionality as you were used to you just need to set the "Activity duration" field instead of the "Initialy planned hours".

**Comment by sysadminmatmoz** [comment]
There's another thing to consider:
In the tasks (activities) you have 2 planned times (task form, tab other settings).

- Activity duration - planned calendar time, like, I plan we'll finish in 40 hours (5 days) from start, but it doesn't mean, that we'll work for 40 hours - **TIME PLANNING**

- Initially  Planned Hours - cost man hours planned for that activity, like - even, if we'll finish it in 40 hours, we'll actually invest only 5 hours of real work (like 1 hr every day) - **COST PLANNING**

The Limitations section in that same tab defines the initial calculation data - fill only one of them, depending on your limitations for that task:

- earliest start date - we can't start before that date, but we can start on that date or later

- latest finish date - do whatever you want, but we have to finish until then
Based on those limits (and task relationships - predecessor and successor tasks) and the duration field the button calculate network calculates the optimal dates for the entire task network (a group of interconnected tasks).
